blueprints/replays.py

```diff
@@ -96,7 +96,7 @@ def view_replay(id_):
                                item_dict=get_item_dict())
 
 
-@bp.route('/parsed/view/<id_>/positions')
+@bp.route('/parsed/view/positions/<id_>') # ID must always be last
```

Comment:
The reason the ID wasn't at the end of the URL was because in JS it is easy to just do:
```javascript
$.getJSON('positions/', function (data) { ...
```
If the ID is placed at the end of the array, the URL builder needs knowledge of the current replay ID, and it is less page agnostic. That was the only concern. Typically sub-pages place their info at the end of the URL instead of making a whole new root.

Reply:
Also petition to change the name from positions to something else. That was the first name I came up with a month ago, and I'm infamous for naming things terribly.

Reply:
I think that name is okay.  But the problem was that the URL was throwing 404s and stopped when I put the id at the end